### PR TITLE
hint

### PR DESCRIPTION
**Pull request by noahbarger**

### CODE REVIEW

**Review by SteamsDev — APPROVED** [review]

File: MVC/view/macGUI.py [hunk 1]
```diff
@@ -27,14 +31,10 @@ def __init__(self, parent, ctrl):
         self.parent.config(menu=self.menu)
         self.img.place(x = 0,y = 0)
         # created an input box
-        self.inputFrame = tk.Frame(self.parent)
```
Comment (tmhughes1) [inline MVC/view/macGUI.py:30]: Why was this deleted? Please add it back!

Comment (tmhughes1) [reply MVC/view/macGUI.py:30]: Unless you deleted it for a reason but please specify if so!

Comment (auanmu) [reply MVC/view/macGUI.py:30]: also deleted more than just this line check lines 30-35, and 37
